server/package.json

```diff
@@ -4,10 +4,12 @@
   "main": "index.js",
```

Comment:
These are changes for backend in #7 which shouldn't also be a part of this pr. It probably is because you branch out from `tim/backend-architecture` when creating this branch. We should always branch out from `main` (unless intentionally).

At this point, you would want #7 to be approved and merged before this pr. Or you would reset/force-push/cherry-pick commits to the state where the commit doesn't exist in the branch. Many ways to resolve.

Reply:
Yes I didn't realize i branched from that branch by accident, I also put (backend) in a few commits that were frontend. Brain.exe has stopped working.